manifest.yml

```diff
@@ -1,12 +1,10 @@
 manifest_version: 1
-name: ImageNormalizer
-kind: crafter
-type: pod
-description: '`ImageNormalizer` resizes and normalizes images'
+name: ImagePaddlehubEncoder
+kind: encoder
```

Comment:
`kind` doesn't exist in the new manifest spec https://docs.google.com/document/d/1mLNkLlg4aiwhHmM0Nq-ndzbI8cTUQH0Ub8qgtnPqsFI/edit#heading=h.cz2166romzp